apps/website/package.json

```diff
@@ -29,92 +29,90 @@
   },
   "dependencies": {
     "@alveusgg/data": "github:alveusgg/data",
-    "@aws-sdk/client-s3": "^3.312.0",
-    "@aws-sdk/s3-request-presigner": "^3.312.0",
-    "@headlessui/react": "^1.7.14",
-    "@heroicons/react": "^2.0.17",
-    "@next-auth/prisma-adapter": "^1.0.5",
-    "@next/env": "^13.3.0",
+    "@aws-sdk/client-s3": "^3.345.0",
+    "@aws-sdk/s3-request-presigner": "^3.345.0",
+    "@headlessui/react": "^1.7.15",
+    "@heroicons/react": "^2.0.18",
+    "@next-auth/prisma-adapter": "^1.0.7",
+    "@next/env": "^13.4.4",
     "@paralleldrive/cuid2": "^2.2.0",
-    "@prisma/client": "^4.12.0",
-    "@tanstack/react-query": "^4.29.1",
-    "@trpc/client": "^10.20.0",
-    "@trpc/next": "^10.20.0",
-    "@trpc/react-query": "^10.20.0",
-    "@trpc/server": "^10.20.0",
+    "@prisma/client": "^4.15.0",
+    "@tanstack/react-query": "^4.29.12",
+    "@trpc/client": "^10.29.0",
+    "@trpc/next": "^10.29.0",
+    "@trpc/react-query": "^10.29.0",
+    "@trpc/server": "^10.29.0",
     "@vercel/analytics": "^1.0.1",
+    "canvas-hypertxt": "^1.0.1",
```

Comment:
this is for generating social media images which is incomplete and behind a feature flag right now

Reply:
Have you looked at using Vercel's satori library?

Reply:
Yeah i did. There were a few things not working yet, but I don't remember what exactly. I can remove the code from the PR and we can revisit later if and when we pursue twitter/ig?!